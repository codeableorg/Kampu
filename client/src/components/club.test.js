import React from "react";
import { render, fireEvent } from "@testing-library/react";
import { Provider } from "react-redux";
import store from "../store";
import Club from "./club";

const club = {
  id: 1,
  address: "Jr cayumba 440",
  favorited: false,
  favorited_count: 2,
  image: "some_picture.jpg",
  name: "Club #1"
};

test("Club component", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Club club={club} />
    </Provider>
  );

  expect(asFragment()).toMatchSnapshot();
});
