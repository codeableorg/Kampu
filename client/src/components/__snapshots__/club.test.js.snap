// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Club component 1`] = `
<DocumentFragment>
  .emotion-4 {
  background: white;
  border-radius: .5em;
  box-shadow: 0px 2px 5px 0px rgba(0,0,0,.12);
  box-sizing: border-box;
  padding: 2rem;
  width: 100%;
  margin-top: 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 1rem;
}

.emotion-0 {
  width: 50px;
  height: 50px;
  background: #ddd;
  border-radius: 50%;
  margin-right: 1em;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-1 {
  cursor: pointer;
}

.emotion-2 {
  font-size: 14px;
  font-weight: bold;
  color: #657786;
  margin-left: 5px;
}

<div
    class="emotion-4"
    role="listitem"
  >
    <div
      class="emotion-0"
    />
    <div>
      <h3>
        Club #1
      </h3>
      <div>
        Address: Jr cayumba 440
      </div>
      <div>
        Price: 
      </div>
      <div
        class="emotion-3"
      >
        <svg
          class="emotion-1"
          fill="none"
          height="18px"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="18px"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z"
          />
        </svg>
        <span
          class="emotion-2"
        >
          2
        </span>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
